how-to/running.rst

```diff
@@ -417,6 +419,23 @@ the bridge interface on your computer):
     sudo ufw allow in on lxdbr0
     sudo ufw route allow in on lxdbr0
 
+.. _building-on-arm:
+
+Building on ARM
+---------------
+
+When running ``make schema``, installing ``pynacl`` can run into `a bug <https://github.com/pyca/pynacl/issues/553>`_ when building for an ARM architecture, leading to ``ERROR: Failed building wheel for pynacl``.
+
+The following needs to be run when having this issue:
```

Comment:
nit: I'd word it with more of a `If you encounter this issue, try running the following` or something more along those lines

Reply:
I'd prefer it if the instructions can be updated to build the local `sdist` of `PyNaCl` instead of `pip install`'ing it from the internetl.